packages/webpack/src/runtime/runtime.js

```diff
@@ -24,8 +26,23 @@ if (LOCKDOWN_ON) {
   )
 }
 
+const knownWritableFields = new Set()
+
+values(LAVAMOAT.policy.resources).forEach((resource) => {
+  if (resource.globals && typeof resource.globals === 'object') {
+    entries(resource.globals).forEach(([key, value]) => {
+      if (value === 'write') {
+        knownWritableFields.add(key)
+      }
+    })
+  }
+})
+
 const { getEndowmentsForConfig, copyWrappedGlobals } =
-  LAVAMOAT.endowmentsToolkit()
+  LAVAMOAT.endowmentsToolkit({
+    handleGlobalWrite: true,
+    knownWritableFields,
```

Comment:
This is all that needed to change in usage to support global store now. Should work the same for endomoat.